app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    comment = Comment.new(comment_params)
+    comment.author = current_user
+    comment.post = Post.find(comment_params[:post_id])
```

Comment:
- [Optional] Once more, we can further reduce these lines of code to: `@comment = @current_user.comments.new(comment_params) `